### PR TITLE
Feature/170 calculate and fetch cutsets

### LINKED ISSUE #170: Add button to execute fault tree cutset calculation and fetch the result.

### PR DESCRIPTION
**Pull request by kostobog**
Fixes #170

### CODE REVIEW

File: src/components/editor/faultTree/Editor.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ const Editor = ({setAppBarName}: DashboardTitleProps) => {
 
     useEffect(() => {
         if (faultTree) {
+            console.log(faultTree.faultEventScenarios)
```
Comment (blcham) [inline src/components/editor/faultTree/Editor.tsx:48]:
```suggestion
```

Comment (blcham) [reply src/components/editor/faultTree/Editor.tsx:48]: I do not consider it a good idea to include console logs.

File: src/components/editor/faultTree/Editor.tsx [hunk 2]
```diff
@@ -146,6 +148,17 @@ const Editor = ({setAppBarName}: DashboardTitleProps) => {
         })
     }
 
+    const handleCutSetAnalysis = () => {
+        console.log(`handleCutSetAnalysis(${faultTree.iri})` )
```
Comment (blcham) [inline src/components/editor/faultTree/Editor.tsx:152]:
```suggestion
```

Comment (blcham) [reply src/components/editor/faultTree/Editor.tsx:152]: I do not consider it a good idea to include console logs.

**Review by blcham — APPROVED** [review]
LGTM, consider removing console.logs.

File: src/components/editor/faultTree/Editor.tsx (hunk 2 above)
Comment (kostobog) [reply src/components/editor/faultTree/Editor.tsx]: resolved

File: src/components/editor/faultTree/Editor.tsx (hunk 1 above)
Comment (kostobog) [reply src/components/editor/faultTree/Editor.tsx]: resolved

**Comment by blcham** [comment]
LGTM, can be merged.

**Pull request merged**